query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/ServicesQueries.java

```diff
@@ -53,6 +55,66 @@ static QueryRequest buildQuery1() {
             .build());
 
 
+    builder.addGroupBy(
+        Expression.newBuilder().setColumnIdentifier(serviceId));
+    builder.addGroupBy(
+        Expression.newBuilder().setColumnIdentifier(serviceName));
+    Function serviceDurationPercentileFunc = Function.newBuilder().addArguments(
+        Expression.newBuilder().setColumnIdentifier(ColumnIdentifier.newBuilder().setColumnName("SERVICE.duration").build()).build()).setFunctionName("PERCENTILE99")
+        .build();
+    OrderByExpression orderByExpression = OrderByExpression.newBuilder().setOrder(SortOrder.DESC).setExpression(
+        Expression.newBuilder().setFunction(serviceDurationPercentileFunc).build()).build();
+    builder.addOrderBy(orderByExpression);
+    builder.setLimit(10000);
+
+    return builder.build();
+  }
+
+  static QueryRequest buildAvgRateQuery() {
+    Builder builder = QueryRequest.newBuilder();
+    ColumnIdentifier serviceId = ColumnIdentifier.newBuilder().setColumnName("SERVICE.id").build();
```

Comment:
ColumnIdentifier.newBuilder().setColumnName("SERVICE.id").build() can be static method and have a meaningful name.

Reply:
Yes can be done and added to separate utils class.